examples/react/testing-vitest/vitest.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from "vitest/config";
+import react from "@vitejs/plugin-react";
+import path from "node:path";
+
+export default defineConfig({
+  plugins: [react()],
+  test: {
+    globals: true,
```

Comment:
[MEMO] It doesn't seem like this is applied properly, since I still needed to implicitly import the vitest members. 

Will need further investigation.

Reply:
Thank you for the suggestion! It looks like it's resolved with just `"types" : ["vitest/globals"]`.
Adding `"node"` to the list causes an error saying that it cannot find the type definition for "node".